### PR TITLE
[Custom] Update custom db-libs

### PR DESCRIPTION
**Pull request by rbankston**
Updated custom db-libs ported from the v4 custom chef recipes for no-db envs

### CODE REVIEW

**Comment by radamanthus** [comment]
@rbankston how will end-users use this recipe? Is this automatically enabled, or do they have to add `include_recipe 'db-client-libs'` to `ey-custom/recipes/after-main.rb` ?

File: cookbooks/db_client_libs/recipes/default.rb [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+  #    - MySQL: /engineyard/portage/engineyard/dev-db/percona-server/*.ebuild
+install_packages=[
+  {
+    :server => 'postgresql-server',   # postgresql-server or percona-server
+    :version => '9.4.4',
+    :default_slot => false,                   # postgres packages only, use true only with no_db environments, makes this package the system default
+  },
+  ]
+
+if ['app', 'app_master', 'util'].include?(node['instance_role'])
```
Comment (dvalfre) [inline cookbooks/db_client_libs/recipes/default.rb]: it should be `node['dna']['instance_role']`

**Comment by dvalfre** [comment]
Added it as a custom chef the way Rad mentioned, for a customer that needed it asap.  It works as intended.
In any case, should rename it `ey-db-client-libs` to make clear it's managed by EY?

**Comment by tpoland** [comment]
Adding the ey would be fine. We are intending to remove this once the db_provider feature rolls out. That feature still requires a DNA update.

**Pull request merged**